site/_scss/site/objects/_components.scss

```diff
@@ -0,0 +1,142 @@
+.grid.three {
```

Comment:
@amymanion as with others, break this into separate partials for each "component"